### ISSUE TITLE
请教下两阶段预训练的schedule设置的细节

### ISSUE THREAD

**Issue by hitvoice** [ymcui/Chinese-BERT-wwm#106]
论文中写到：
> We train 100K steps on the samples with a maximum length of 128, batch size of 2,560, an initial learning rate of 1e-4 (with warm-up ratio 10%). Then, we train another 100K steps on a maximum length of 512 with a batch size of 384 to learn the long-range dependencies and position embeddings.

请教一下这样两阶段训练时，lr schdule是下面哪一种？
1. warmup 10k steps到1e-4，再用190k steps线性衰减到0，中途第100k step的时候换了最大长度；
2. warmup 10k steps到1e-4，经过90k steps线性衰减到0，接下来换最大长度以后用10k steps上升到1e-4，最后90k steps线性衰减到0。

如果是第二种，第二阶段的预训练loss是否会出现先升再降的情况？

**Comment by ymcui** [comment]
是第二种情况，具体的loss的变化我得再找找。

**Comment by hitvoice** [comment]
好的，谢谢🙏

**Comment by ymcui** [comment]
大体趋势上，在第二阶段loss是会先上升后下降的。

**Comment by hitvoice** [comment]
好的，感谢！❤️
